util/make_timeline.py

```diff
@@ -97,14 +135,30 @@ def parse_file(args):
 
     entries = []
     started = False
-
+    encounter_sets = []
     with args.file as file:
+        # If searching for encounters, divert and find start/end first3
+        if args.search_fights:
+            encounter_sets = find_fights_in_file(file)
+            # If all we want to do is list encounters, stop here and give to the user.
+            if args.search_fights < 0:
+                return [f'{i + 1}. {e_info[0]} {e_info[1]} {e_info[2]}' for i, e_info in enumerate(encounter_sets)]
```

Comment:
Do the items in encounter_set have more than 3 attributes? If not, you can do:
```python
" ".join(e_info)
```
instead of enumerating them. Or, we can make them their own class with separate attributes if you think that's cleaner, like Encounter with `start_time`, `end_time`, and `name`.

Reply:
You're right, it's a guaranteed 3 elements. I appreciate this.